### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by natebosch**
Add a separate template for reporting a bug or requesting a feature with
the test runner.
Make the language about `package:checks` specific that the package is in
preview and we are taking any type of feedback.

### CODE REVIEW

**Review by kevmoo — APPROVED** [review]
LGTM

File: .github/ISSSUE_TEMPLATE/99_other.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (devoncarew) [inline .github/ISSSUE_TEMPLATE/99_other.md:1]: I don't think we need this template; unless you disable it, people will see an 'open generic issue' at the bottom of the list of templates.

File: .github/ISSSUE_TEMPLATE/01_test_runner_bug.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: I found a bug in the test runner
+about: >-
+  Report a bug in running tests, integration with a specific platform, or
+  any behavior of 'package:test'.
+title: ''
+labels: bug
```
Comment (devoncarew) [inline .github/ISSSUE_TEMPLATE/01_test_runner_bug.md:7]: Just for clarity - you prefer not to have a `package:test` label applied here?

**Review by devoncarew — APPROVED** [review]
lgtm (though I believe we can remove one of the templates)

File: .github/ISSSUE_TEMPLATE/01_test_runner_bug.md (hunk 2 above)
Comment (natebosch) [reply .github/ISSSUE_TEMPLATE/01_test_runner_bug.md:7]: Yes, I'd be fine adding a `needs-triage` label or something if we are worried about the `bug` label potentially causing it to get lost.

My idea is that the user reports the bug, and the author determines which part of the implementation is relevant.

**Pull request merged**